Makefile

```diff
@@ -83,11 +83,11 @@ poetry-shell:
 
 # Run evcc with python
 run-evcc:
-	$(shell which python) iso15118/evcc/main.py $(config)
+	poetry run python iso15118/evcc/main.py $(config)
```

Comment:
I had some issues in the past running it like that in a Mac.
Can someone confirm this? @mdwcrft @snorkman88

Reply:
This works for me